app/src/main/java/com/x8bit/bitwarden/data/platform/manager/FeatureFlagManager.kt

```diff
@@ -0,0 +1,51 @@
+package com.x8bit.bitwarden.data.platform.manager
+
+import kotlinx.coroutines.flow.StateFlow
+
+/**
+ * Manages the available feature flags for the Bitwarden application.
+ */
+interface FeatureFlagManager {
+    /**
+     * Returns a map of constant feature flags that are only used locally.
+     */
+    val featureFlagsLocal: Map<String, Boolean>
```

Comment:
I'm curious why the interfaces exposes both this map and the flow of a map from the server. When the rest of the app interacts with the feature flags, it shouldn't care whether the flag is local or from the server, right?

Reply:
Maybe it's a naming confusion. Local is not meant as cached, the local flags are just constants that live on the device and are not affected by server/region changes. 
Server flags are provided by the server, are dynamic and can change over time.
For now locals are only used to initialise the SDK.

Reply:
These flags are currently just used for the SDK. So we could call it `sdkFeatureFlags` or something like that?

Reply:
For the time being I think we can. If, down the line, we need constant flags locally we can add them as such.